boxes/README.md

```diff
@@ -0,0 +1,34 @@
+# Automated Vagrant box generation
+
+This section defines the procedure to build Vagrant boxes required for WSO2 Identity Server single node
```

Comment:
Shall we mention the IS version here since we have specified the version in the scripts?

Reply:
+1 Yes, that's a good thought!